pyproject.toml

```diff
@@ -18,7 +18,7 @@ celery = "^5.2.3"
 pyproj = "^3.6.1"
 sentry-sdk = "^1.45.1"
 enmerkar = "^0.7.1"
-ncdjango = { git = "https://github.com/consbio/ncdjango.git", branch = "main" }
+ncdjango = { git = "https://github.com/consbio/ncdjango.git", tag = "1.3.7" }
```

Comment:
This doesn't need to be installed from the repo anymore. Version 1.3.7 is on PYPI.

Reply:
Same for trefoil. Version 0.4.0 is on PYPI.

Reply:
Gotcha. Updated